addons/zebra-stripes/addon.json

```diff
@@ -0,0 +1,21 @@
+{
+  "$schema": "https://raw.githubusercontent.com/ScratchAddons/manifest-schema/dist/schema.json",
+  "name": "Nested blocks zebra stripes",
+  "description": "Nested blocks become zibra striped.",
```

Comment:
Zibra

Also what does "zebra striping" even mean?

Reply:
Basically blocks in another block with the same category alternate between becoming light and dark